composeApp/src/commonMain/kotlin/com/github/familyvault/services/ChatService.kt

```diff
@@ -25,7 +26,7 @@ class ChatService(
     private val imagePickerService: IImagePickerService,
 ) : IChatService {
     override suspend fun createGroupChat(
-        name: String, members: List<FamilyMember>
+        name: String, members: List<FamilyMember>, groupChatIcon: ChatIconType?
```

Comment:
I niżej też tak zrób

```suggestion
        name: String, members: List<FamilyMember>, chatIcon: ChatIconType?
```